ansible/roles/service/hippo_authoring/files/hippo_authoring.yml

```diff
@@ -0,0 +1,280 @@
+          PropagateAtLaunch: true
+      VPCZoneIdentifier:
+        Fn::Split:
+          - ","
+          - Fn::ImportValue: !Sub "${Environment}-vpc::SubnetsPrivate"
+    CreationPolicy:
+      ResourceSignal:
+        Count: 0
+        Timeout: PT10M
+    UpdatePolicy:
+      AutoScalingRollingUpdate:
+        MinInstancesInService: 1
+        MaxBatchSize: 1
+        PauseTime: PT10M
+        WaitOnResourceSignals: true
+
+  ##
+  # Loadbalancing
+  #
+  LoadBalancerSg:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Enable acces from VPC
+      VpcId:
+        Fn::ImportValue: !Sub "${Environment}-vpc::VpcId"
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: 80
+          ToPort: 80
+          CidrIp: 156.67.242.53/32
+        - IpProtocol: tcp
+          FromPort: 443
+          ToPort: 443
+          CidrIp: 156.67.242.53/32
+      Tags:
+        - Key: Name
+          Value: !Sub "${Environment}-hippo-authoring"
+
+  LoadBalancer:
+    Type: AWS::ElasticLoadBalancing::LoadBalancer
+    Properties:
+      ConnectionDrainingPolicy:
+        Enabled: true
+        Timeout: 30
+      CrossZone: true
+      # HealthCheck:
+      #   Target: HTTP:8080/health-check
+      #   HealthyThreshold: 2
+      #   UnhealthyThreshold: 3
+      #   Interval: 10
+      #   Timeout: 2
+      Listeners:
+        - LoadBalancerPort: 80
+          Protocol: HTTP
+          InstancePort: 8080
+          InstanceProtocol: HTTP
+        # - LoadBalancerPort: 443
+        #   Protocol: HTTPS
+        #   SSLCertificateId: !Ref SSLCertificateId
+        #   InstancePort: 8080
+        #   InstanceProtocol: HTTP
+      Scheme: internet-facing
+      SecurityGroups:
+        - !Ref LoadBalancerSg
+      Subnets:
+        Fn::Split:
+          - ","
+          - Fn::ImportValue: !Sub "${Environment}-vpc::SubnetsPublic"
+      Tags:
+        - Key: Name
+          Value: !Sub "${Environment}-hippo-authoring"
+
+  ELBDNSRecordPublic:
+    Type: AWS::Route53::RecordSet
+    Properties:
+      Comment: Hippo Authoring public
+      HostedZoneId:
+        Fn::ImportValue: !Sub "${Environment}-public-dns::ZoneId"
+      Name: !Sub "hippo-authoring.web.${HostedDomainPublic}"
+      TTL: 300
+      Type: CNAME
+      ResourceRecords:
+        - Fn::GetAtt:
+          - LoadBalancer
+          - DNSName
+
+  ##
+  # Scaling Up and Down
+  #
+  # Scaling up should double ASG size. Scaling down will take 1 node at a time every
+  # 10 minutes.
+  #
+  ScaleUpPolicy:
+    Type: AWS::AutoScaling::ScalingPolicy
+    Properties:
+      AdjustmentType: PercentChangeInCapacity
+      AutoScalingGroupName: !Ref ASGroup
+      Cooldown: 180
+      ScalingAdjustment: 100
+
+  ScaleDownPolicy:
+    Type: AWS::AutoScaling::ScalingPolicy
+    Properties:
+      AdjustmentType: ChangeInCapacity
+      AutoScalingGroupName: !Ref ASGroup
+      Cooldown: 600
+      ScalingAdjustment: -1
+
+  CPUAlarmHigh:
```

Comment:
Do we actually *want* to autoscale? Remember, we'll have a fixed and limited number of licences. I'd be tempted not to autoscale based on server load.

Reply:
$$ savings 2 servers for more then 12h a day VS 6 servers 24/7 makes a difference in long run.